### ISSUE TITLE
export default named generators

### ISSUE THREAD

**Issue by u9520107** [6to5/6to5#609]
Currently if mod.js has 
```
export default function mod() {};
```
The transpiled code would be: 
```
...
module.exports = mod;
function mod(){}
```
However, if gen.js is:
```
export default function *gen() {};
```
The result is:
```
...
var gen = regeneratorRuntime.mark(....);
```
The transpiled code is missing module.export = gen;

**Comment by u9520107** [comment]
I should note that if the gen.js is modified to be as such:
```
function *gen() {};
export default gen;
```
Then the transpiled code does have module.export set to it.

**Comment by sebmck** [comment]
Looks like regenerator is incorrectly removing the `ExportDeclaration` and replacing it with just the `VariableDeclaration`. I'll look into it now, *should* be a simple fix.

**Comment by sebmck** [comment]
Fixed as of 3.0.8, thanks!

**Comment by u9520107** [comment]
@sebmck Whoah.. that was awesomely fast!!. Thx.
